### PR TITLE
Integrate inferred cell type and cluster marker gene heatmap

### LINKED ISSUE #441: Marker genes shown in marker gene plot on first landing not consistent with gene shown when single cluster view selected
Marker genes shown in marker gene plot on first landing not consistent with gene shown when single cluster view selected.

Example experiment: https://wwwdev.ebi.ac.uk/gxa/sc/experiments/E-MTAB-10371/ 

Step to reproduce:
1. Go to `Marker Genes` tab
2. Check the genes for cluster 2
3. The filtered marker gene heatmap is not the same set of genes as shown in the full heatmap

### PR DESCRIPTION
**Pull request by lingyun1010**
Please refer to the related issues for detailed description. https://github.com/ebi-gene-expression-group/atlas-web-single-cell/issues/441

### CODE REVIEW

File: app/src/main/javascript/bundles/experiment-page/html/index.html [hunk 1]
```diff
@@ -326,7 +326,7 @@
     ];
 
     experimentPageDemo.render({
-        atlasUrl: 'https://wwwdev.ebi.ac.uk/gxa/sc/',
+        atlasUrl: 'http://localhost:8080/gxa/sc/',
```
Comment (upendrakumbham) [inline app/src/main/javascript/bundles/experiment-page/html/index.html:329]: Maybe this needs to be  updated to `wwwdev`

**Review by upendrakumbham — CHANGES_REQUESTED** [review]
move back `localhost ` to `wwwdev`

File: app/src/main/javascript/bundles/experiment-page/html/index.html (hunk 1 above)
Comment (lingyun1010) [reply app/src/main/javascript/bundles/experiment-page/html/index.html:329]: Actually it doesn't matter, I would like to test the localhost actually. And normally we should test with local not the production environment.

**Review by ke4 — DISMISSED** [review]
LGTM, but there are 4 broken tests 😢

**Pull request merged**